<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Position</class>
 <widget class="QDialog" name="Position">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Штатное расписание</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Штатное расписание</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3">
       <widget class="QTableView" name="positionsTableView"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>275</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Добавить единицу</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Служба</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="deptsComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Группа</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="groupsComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Должность (полн.)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="posLongName"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Должность (кратк.)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="posShortName"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Position</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Position</receiver>
   <slot>delPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Position</receiver>
   <slot>addPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Position</receiver>
   <slot>cancelAddPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>delPosition()</slot>
  <slot>addPosition()</slot>
  <slot>cancelAddPosition()</slot>
 </slots>
</ui>
